workloads/hyper-scale/common.sh

```diff
@@ -4,22 +4,28 @@ source env.sh
 
 prep(){
     if [[ -z $(go version) ]]; then
-        curl -L https://go.dev/dl/go1.17.6.linux-amd64.tar.gz -o go1.17.6.linux-amd64.tar.gz
-        tar -C /usr/local -xzf go1.17.6.linux-amd64.tar.gz
+        curl -L https://go.dev/dl/go1.18.2.linux-amd64.tar.gz -o go1.18.2.linux-amd64.tar.gz
```

Comment:
We might want to add make, jq, kubectl/oc as dependencies to the readme. Thoughts?

Reply:
updated readme for make and jq, include cli installs in the script.

Reply:
Can we consider leveraging `-o jsonpath`, with no additional dependencies on jq, Thoughts?